packages/react-components/react-menu/etc/react-menu.api.md

```diff
@@ -295,19 +297,25 @@ export type MenuState = ComponentState<MenuSlots> & Pick<MenuProps, 'defaultChec
 export const MenuTrigger: React_2.FC<MenuTriggerProps> & FluentTriggerComponent;
 
 // @public
-export type MenuTriggerChildProps = Required<Pick<React_2.HTMLAttributes<HTMLElement>, 'onClick' | 'onMouseEnter' | 'onMouseLeave' | 'onMouseMove' | 'onContextMenu' | 'onKeyDown' | 'aria-haspopup' | 'id'>> & {
-    ref?: React_2.Ref<never>;
-    'aria-expanded': boolean | undefined;
-};
+export type MenuTriggerChildProps<Type extends ARIAButtonType = ARIAButtonType, Props = {}> = ARIAButtonResultProps<Type, Props & {
+    'aria-haspopup': 'menu';
+    'aria-expanded'?: boolean;
+    id: string;
+    ref: React_2.Ref<never>;
+    onClick: React_2.MouseEventHandler<HTMLButtonElement & HTMLAnchorElement & HTMLDivElement>;
+    onKeyDown: React_2.KeyboardEventHandler<HTMLButtonElement & HTMLAnchorElement & HTMLDivElement>;
```

Comment:
why are `onClick` and `onKeyDown` added here ? aren't they returned by `useAriaButton` ? is it just so that the html element in the event types are set correctly ?

Reply:
They're added on `useARIAButton` but as possibly `undefined`, this specifies that they'll be present by the trigger's side at least